src/contributing/void-docs/infradocs.md

```diff
@@ -0,0 +1,9 @@
+# InfraDocs
+
+[InfraDocs](https://infradocs.voidlinux.org/) is the manual for Void systems
+management.
+
+The manual addresses the technical infrastructure managed by different systems
+and how the project works on a human level.
+
+Knowledge of Void or base Linux administration is assumed.
```

Comment:
This is assumed across most if not all of Void, and probably doesn't need to be explicitly stated.